db/version_set.cc

```diff
@@ -2028,7 +2032,7 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
         get_perf_context()->per_level_perf_context_enabled;
     StopWatchNano timer(clock_, timer_enabled /* auto_start */);
     *status = table_cache_->Get(
-        read_options, *internal_comparator(), *f->file_metadata, ikey,
+        read_opts, *internal_comparator(), *f->file_metadata, ikey,
```

Comment:
We should be using the user provided read_options.

Reply:
will update.